### DISCUSSION TITLE
Programming

### DISCUSSION THREAD

**Discussion by streetprophet90** [streetprophet90/Synergy-Codes#11] [Q&A] [answered]
What is Programming?

**Comment by streetprophet90** [comment, marked as answer]
I'll it's simply talking to your computer

**Reply by streetprophet90** [reply]
With a programming language

**Reply by deedadey-dd** [reply]
Programming is what we are doing now

**Comment by The-Leo** [comment]
Programming is a technological process of talking to your computer to solve problems.

**Reply by The-Leo** [reply]
To do that, you need a programming language

**Reply by deedadey-dd** [reply]
I prefer an easy to use language because I believe the core of programming has to do with your creativity in solving issues and not really a thorough understanding of the language being coded in.
My 2 cents.

**Comment by streetprophet90** [comment]
What are some programming languages available

**Comment by deedadey-dd** [comment]
I prefer an easy to use language because I believe the core of programming has to do with your creativity in solving issues and not really a thorough understanding of the language being coded in.
My 2 cents.
